src/components/TagList/TagList.jsx

```diff
@@ -0,0 +1,17 @@
+import { Tag, TagList } from "../../pages/Admin/Components/FormInput/TagInputStyled"
+
+export default function TagMapper({tags, action}) {
+
+    
+    if(tags) return (
+        <TagList>
+        {tags.map(tag=>(
+            <Tag key={tag}>
```

Comment:
how about we use index number instead? Although we should always use a unique id for key.